airbyte-integrations/connectors/source-postgres/src/main/java/io/airbyte/integrations/source/postgres/PostgresSource.java

```diff
@@ -81,6 +83,8 @@ public class PostgresSource extends AbstractJdbcSource<PostgresType> implements
   private static final Logger LOGGER = LoggerFactory.getLogger(PostgresSource.class);
   private static final int INTERMEDIATE_STATE_EMISSION_FREQUENCY = 10_000;
 
+  public static final String ACTIVITY_TRACE_OPERATION_NAME = "activity";
```

Comment:
This is the name of the operations in the workers, which are Temporal activities.  I suspect that we will want a different operation name for the traces in here.  I'm not sure what we call the operations performed by a source, but whatever it is, that is what we should use for the operation name instead of activity.